CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+[1.x CHANGELOG](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md)
```

Comment:
This works. Minor suggestion:

Looking for older entries? See the [1.x CHANGELOG](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).